internal/pkg/template/template_integration_test.go

```diff
@@ -435,15 +436,16 @@ func TestTemplate_ParseNetwork(t *testing.T) {
      - Fn::ImportValue: !Sub '${AppName}-${EnvName}-EnvironmentSecurityGroup'
      - "sg-1bcf1d5b"
      - "sg-asdasdas"
+	 - Fn::ImportValue: mydb-sg001
 `,
 		},
 		"should render AWS VPC configuration without default environment security group": {
 			input: template.NetworkOpts{
 				AssignPublicIP: "DISABLED",
 				SubnetsType:    "PrivateSubnets",
-				SecurityGroups: []string{
-					"sg-1bcf1d5b",
-					"sg-asdasdas",
+				SecurityGroups:[]template.SecurityGroup{
+					template.plainSecurityGroup("sg-1bcf1d5b"),
+					template.plainSecurityGroup("sg-asdasdas"),
```

Comment:
wat 🤯 am I dreaming but how does this work, isn't this type private?? I would have expected
```suggestion
					template.PlainSecurityGroup("sg-1bcf1d5b"),
					template.PlainSecurityGroup("sg-asdasdas"),
```